dev/devicelab/lib/tasks/run_tests.dart

```diff
@@ -11,24 +11,121 @@ import '../framework/framework.dart';
 import '../framework/task_result.dart';
 import '../framework/utils.dart';
 
+TaskFunction createAndroidRunReleaseTest() {
+  return AndroidRunOutputTest(release: true);
+}
+
 TaskFunction createMacOSRunReleaseTest() {
   return DesktopRunOutputTest(
     // TODO(cbracken): https://github.com/flutter/flutter/issues/87508#issuecomment-1043753201
     // Switch to dev/integration_tests/ui once we have CocoaPods working on M1 Macs.
     '${flutterDirectory.path}/examples/hello_world',
     'lib/main.dart',
     release: true,
+    allowStderr: true,
   );
 }
 
+class AndroidRunOutputTest extends RunOutputTask {
+  AndroidRunOutputTest({required super.release}) : super(
+    '${flutterDirectory.path}/dev/integration_tests/ui',
+    'lib/main.dart',
+  );
+
+  @override
+  Future<void> prepare(String deviceId) async {
+    // Uninstall if the app is already installed on the device to get to a clean state.
+    final List<String> stderr = <String>[];
+    print('uninstalling...');
```

Comment:
Are print statements allowed here?

Reply:
AFAICT yes, there's lots of `print`s throughout the devicelab tasks: https://github.com/search?q=repo%3Aflutter%2Fflutter+path%3Adev%2Fdevicelab%2Flib%2Ftasks+print&type=code